src/ExpressionEvaluator/Core/Test/FunctionResolver/FunctionResolverTest.csproj

```diff
@@ -81,5 +81,8 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
@tmat this tag is on some project files, so not common. Which common file you mean?